tests/Unit/CsatAjaxTest.php

```
@@ -0,0 +1,296 @@
+	public function test_rating_above_range_rejected(): void {
+		$_POST = array(
+			'ticket_id' => '42',
+			'rating'    => '6',
+			'nonce'     => 'good-nonce',
+		);
+		WP_Mock::userFunction( 'wp_verify_nonce', array( 'return' => 1 ) );
+
+		$stop = $this->invoke();
+		$this->assertFalse( $stop->is_success );
+		$this->assertSame( 400, $stop->status );
+	}
```

PHPMD (2.15.0)

[error] test_rating_above_range_rejected accesses the super-global variable $_POST. (undefined)

(Superglobals)